social-media-engagement-template/v0/src/inputs.jsonnet

```diff
@@ -229,69 +337,95 @@
-                name: "Db User",
-                description: "",
-                type: "string",
-                kind: "input",
-                default: "KEBOOLA_WORKSPACE_34417172",
-              },
-              {
-                id: "wr-snowflake-blob-storage-db-warehouse",
-                name: "Db Warehouse",
-                description: "",
+              id: "wr-snowflake-db-host",
+              name: "Hostname",
+              description: "Insert database hostname",
+              type: "string",
+              kind: "input",
+              rules: "required",
+              default: "hostname.com",
+            },
+            {
+              id: "wr-snowflake-db-port",
+              name: "Port",
+              description: "Insert database port number.",
+              type: "string",
+              kind: "input",
+              default: "443",
+              showif: "[wr-snowflake-db-host] != ''",
+            },
+            {
+              id: "wr-snowflake-db-user",
+              name: "Username",
+              description: "Insert database username.",
+              type: "string",
+              kind: "input",
+              default: "userName",
+              showif: "[wr-snowflake-db-host] != ''",
+            },
+            {
+              id: "wr-snowflake-db-password",
+              name: "Database Password",
+              description: "Insert your password to the database.",
+              showif: "[wr-snowflake-db-host] != ''",
+              type: "string",
+              kind: "hidden",
+            },
+            {
+              id: "wr-snowflake-db-database",
+              name: "Database Name",
+              description: "Insert name of your database.",
+              type: "string",
+              kind: "input",
+              default: "DatabaseName",
+              showif: "[wr-snowflake-db-host] != ''",
+            },
+            {
+              id: "wr-snowflake-db-schema",
+              name: "Schema",
+              description: "Insert database schema.",
+              type: "string",
+              kind: "input",
+              default: "Schema",
+              showif: "[wr-snowflake-db-host] != ''",
+            },
+            {
+              id: "wr-snowflake-db-warehouse",
+              name: "Warehouse",
+              description: "Insert database warehouse.",
+              type: "string",
+              kind: "input",
+              default: "Warehouse",
+              showif: "[wr-snowflake-db-host] != ''",
+            },
+          ]
+        },
+        {
+          icon: "component:keboola.wr-google-sheets",
+          name: "Google sheet Destination",
+          description: "Load to data into google sheet",
+          dialogName: "Google Sheet Destination", 
+          dialogDescription: "Make a copy of [this sheet](www.sheet.com) into your Drive. Copy text between 'spreadsheets/d/' and '/edit' and paste it below.",
```

Comment:
zde je potřeba přidat link na ten google sheet, co jsem vytvořila, až to otestuješ a bude to fungovat